pkg/term/windows/windows_test.go

```diff
@@ -0,0 +1,3 @@
+// Dummy file
```

Comment:
@tiborvass I believe you are correct. I've removed the unnecessary windows_test.go file.
@ahmetalpbalkan We had discussed using dummy.go, but we also noted that packages will use a file named after themselves to contain package-level comments. I greatly dislike faux names such as dummy.go because they communicate no information. So...I elected to name it windows.go and include "package" style comments.